storm-client/src/jvm/org/apache/storm/dependency/DependencyUploader.java

```diff
@@ -157,7 +167,14 @@ private boolean uploadDependencyToBlobStore(String key, File dependency)
             AtomicOutputStream blob = null;
             try {
                 blob = getBlobStore().createBlob(key, new SettableBlobMeta(acls));
-                Files.copy(dependency.toPath(), blob);
+//              Files.copy(dependency.toPath(), blob);
+
+                /***
+                 * By default, {@link java.nio.file.Files#copy(Path, OutputStream)} copy bytes only 8k each time,
```

Comment:
I think this comment should be deleted before we merge, as it only makes sense when looking at the current diff.